README.md

```diff
@@ -3,3 +3,4 @@
 Removes need to have an arrow in your inventory to use the Infinity I enchant on your bow.
 
 **Thanks to Myrathi for the original InfiniBows!**
+**Thanks to MeeniMc for the 1.17 port**
```

Comment:
As a general rule of thumb, I think I would leave stuff like this to the repo maintainer to add if they so desire. Some might view this the wrong way. Also, people can see the commit logs and see that you are essentially thanking yourself ;)

Reply:
In my professional setting, the person that makes the change also update the copyrights/authors etc, so that it doesn't get lost/falls on somebody else's plate to update it. Our lawyers care quite a bit that these are maintained accurate so better doing it now than forget later. 

Now this is not the same here, we have no lawyers and everybody is pro-bono, I have no problem removing all this if that rubs you the wrong way :)

Reply:
Huh, really? I guess we hang with different crowds :)

I guess it just seems kinda self-serving to me. I'm sure you didn't mean it that way. I get the professional setting thing, at my workplace there's about 30 copyright notices on each file, haha. But for MC mods, I think the rules are probably a little more lax. I guess my experience comes from someone who updated a FOSS project I used for something, and plastered their name absolutely everywhere, whereas the real authors just had like two sentences in the README. _That_ rubbed me the wrong way.

I guess for me, it's easier to enforce totally objective rules (like "the maintainer will be the one to update credits, etc") than work with more subjective ones (like "oh its okay for you to do it, but not this person because of my own arbitrary ideas on what constitutes creditable work"). Might make some more people upset, but they can't argue with it because there's little to no subjectivity involved.

So, TLDR, I would appreciate the removal of this line. Not because I'm trying to discredit your work. Because I'm weird :)